libr/anal/fcn.c

```diff
@@ -386,6 +386,7 @@ R_API RAnalFunction *r_anal_fcn_new() {
 	fcn->fingerprint = NULL;
 	fcn->diff = r_anal_diff_new ();
 	fcn->has_changed = true;
+	fcn->rbp_as_frame_ptr = true;
```

Comment:
RAnalFunction is an arch that shouldnt hold arch-specific stuff, what im worried about this PR iss that is introducing some x86-specific naming in here. I bet this issue can be replicated on other archs, so i would probably use a different naming for this variable. and in fact im thinking in adding attribute binary mask field in there to hold info like this:

fcn->attributes = (BPFRAME | NORETURN | INLINE | UNUSED | ...)

can you rename this variable tp bpframe? instead of "rbp_as_frame_ptr" ? we will think about having this extended attribuet field later

Reply:
Thank you for your comment. I changed it. I was also thinking about moving can_affect_bp function to op.c.